src/routes/strategic_ecosystems.js

```diff
@@ -38,6 +38,37 @@ const { Router } = require('restify-router');
  *  ]
  */
 
+/**
+ * @apiDefine getSEEAreasExample
+ * @apiSuccessExample {json} Success-Example:
+ *  {
+ *    "national": {
+ *      "area": 123456789,
+ *      "percentage": 0.45,
+ *      "type": "Páramo"
```

Comment:
It will be tested only with this kind of national strategic ecosystem (moor)

Reply:
I don't understand the comment. But in this case "Páramo" is just an example, other ecosystems type are: "Humedal" and "Bosque Seco Tropical"